bot_jobs/get_national_data.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+URL = "https://raw.githubusercontent.com/wiki/djay/covidthailand/combined.csv"
+USE_COL = ["Hospitalized", "Cases", "Deaths"]
+START_DATE = "2021-01-01"
+
+# Get national data from URL
+df = pd.read_csv(URL)
+
+# Select Column and rename
+df = df[["Date",] + USE_COL]
```

Comment:
Also, in case we have some kind of date index, we can set index to date and use `asfreq` to fill in missing dates, then reset the index back. Not sure if this helps, but it's a trick I've just learned yesterday. Might be handy if you want to fill zero values for non-existing dates. (But if it is guaranteed that the input data has no missing dates then this will be useless)

Reply:
Sorry check the latest commit please.

Reply:
Good to know, but the dataset is processed already (You know, not from the government.) So should be perfect.